### DISCUSSION TITLE
WPF samples are completely out of date

### DISCUSSION THREAD

**Discussion by Symbai** [oxyplot/oxyplot#1943] [Q&A] [answered]
https://github.com/oxyplot/documentation-examples/blob/master/HelloWorld/WpfApplication1/MainWindow.xaml

Last modified 8 years ago. There is no `oxy:Plot` or `oxy:LineSeries` in the oxyplot namespace for WPF anymore. Does anyone have a recent very simple example which uses databinding?

**Comment by VisualMelon** [comment, marked as answer]
Sorry about the old docs. These are moved to OxyPlot-Contrib, see https://github.com/oxyplot/oxyplot-contrib/blob/develop/Source/Examples/WPF/WpfExamples/Examples/AreaDemo/MainWindow.xaml for an example.

Relevant nuget package: https://www.nuget.org/packages/OxyPlot.Contrib.Wpf

**Reply by Symbai** [reply]
Thanks, I was coming from the website https://oxyplot.github.io/ and clicked on Documentation then WPF (XAML) which redirected me to the link above. Maybe its a good idea to add a new note to the website that the documentation is outdated?

**Reply by VisualMelon** [reply]
Yes, this has been the plan for about 2 years, I'm just slow and don't really know how all the documentation is built: https://github.com/oxyplot/docs/issues/45

**Reply by Symbai** [reply]
My bad, I'm sorry

**Reply by VisualMelon** [reply]
No, no problem at all: good to have this here so that other people can find it.
